pkg/action/build-uki.go

```diff
@@ -398,7 +398,18 @@ func (b *BuildUKIAction) ukify(sourceDir, artifactsTempDir, cmdline string) erro
 	if err != nil {
 		return fmt.Errorf("running ukify: %w\n%s", err, string(out))
 	}
+
 	b.logger.Debugf("ukify output: %s", string(out))
+
+	// check size of the efi file
+	fi, err := os.Stat(finalEfiName)
+	if err != nil {
+		return fmt.Errorf("getting file info for %s: %w", finalEfiName, err)
+	}
+	if sizeLimit := viper.GetInt64("efi-size-warn"); fi.Size() > sizeLimit*1024 {
```

Comment:
`fi.Size()` is in bytes (https://pkg.go.dev/io/fs#FileInfo). sizeLimit is in Megabytes. To compare I think you need to multiple with `1024` twice.

Reply:
got it. fixed